edenai_apis/features/text/plagia_detection/plagia_detection_dataclass.py

```
@@ -12,25 +11,26 @@ class PlagiaDetectionCandidate(BaseModel):
 
     @root_validator(pre=True)
     def _set_prediction(cls, values: dict) -> dict:
-        plag_score = values.get('plagia_score', None)
+        plag_score = values.get("plagia_score", None)
```

Ruff (0.8.2)

Use `values.get("plagia_score")` instead of `values.get("plagia_score", None)`

Replace `values.get("plagia_score", None)` with `values.get("plagia_score")`

(SIM910)